sample/common/src/commonTest/kotlin/com/example/sqldelight/hockey/SchemaTest.kt

```diff
@@ -4,6 +4,7 @@ import kotlin.test.Test
 import kotlin.test.assertTrue
 
 class SchemaTest : BaseTest() {
+  /*
```

Comment:
does this still need to be commented out?

Reply:
No. I missed these. Just pushed a fix. There's no way do disable running common tests on a specific target apparently so I duplicated the tests (the alternative would be to expect a boolean from each target to decide if we should run the test or not).